lib/plugins/leetcode.js

```diff
@@ -156,7 +157,11 @@ plugin.getProblem = function(problem, cb) {
 
     problem.totalAC = JSON.parse(q.stats).totalAccepted;
     problem.totalSubmit = JSON.parse(q.stats).totalSubmission;
-    problem.desc = he.decode(cheerio.load(q.content).root().text());
+    if (!q.translatedContent) {
+        problem.desc = he.decode(cheerio.load(q.content).root().text());
+    }else{
+        problem.desc = he.decode(cheerio.load(q.translatedContent).root().text());
```

Comment:
can combine into single line like:
```
problem.desc = he.decode(cheerio.load(q.translatedContent || q.content).root().text());
```
assume that q.translatedContent would be null if talking with leetcode.com.